_includes/event-data.html

```diff
@@ -0,0 +1,27 @@
+<td>{{ event.date.start | date: "%A" }}</td>
```

Comment:
This include directive now only works if an `event` variable is in scope. Includes in jekyll can be passed arguments (see [Include](https://jekyllrb.com/docs/includes/)) so that every variable is explicitly passed as an argument.

Keeping it this way makes it very hard to maintain.

A suggestion: `{% include "_includes/event-data.html" event=event %}`, and in this file, replace all `event` occurences with `include.event` so it's passed and used as an argument

Reply:
Furthermore, `event-data` would better be called `event-table-row` so it's more clear that this is a visual include.

Reply:
Refactoring done!